dqc_us_rules/util/facts.py

```diff
@@ -175,15 +175,36 @@ def LegalEntityAxis_facts_by_member(facts):
     results = defaultdict(list)
     for fact in facts:
         legalDim = LEGALENTITYAXIS_DEFAULT
-        dims = [dim for dim in fact.context.segDimValues.values() if dim.isExplicit and dim.member is not None]
-        for dim in dims:
-            if dim.dimension.qname.localName == 'LegalEntityAxis':
-                legalDim = dim.member.qname.localName
-                break
-        results[legalDim].append(fact)
+        if fact_components_valid(fact):
+            dims = [dim for dim in fact.context.segDimValues.values() if dim.isExplicit and dim.member is not None]
+            for dim in dims:
+                if dim.dimension.qname.localName == 'LegalEntityAxis':
+                    legalDim = dim.member.qname.localName
```

Comment:
I will make a ticket for this, but this change isn't even something for this ticket. Github keeps showing everything from other commits has have already been merged meaning that I only touched dqc_us_0006.py and test_dqc_us_0006.py for this ticket, but it is showing a whole bunch of changes that I didn't make.